doc/newsfragments/2901_new.pre_post_hooks_basetest.rst

```diff
@@ -0,0 +1 @@
+Resource hooks ``before_start``, ``after_start``, ``before_stop``, and ``after_stop`` are now supported in ``ProcessRunnerTest`` classes, like ``GTest``, ``Cppunit``, ``HobbesTest``, and ``JUnit``, and also in ``PyTest`` and ``PyUnit``.
```

Comment:
We are supporting these hooks in all tests, it might be a bit confusing to mention ProcessRunnerTest. Maybe something like:

...are now supported in ``Test`` base class, and can be used by GTest, Cppunit, etc.